docs/tutorial-epsilla.rst

```diff
@@ -145,5 +151,7 @@ We can now perform a vector search to find the most relevant project based on a
 
 You will see the returned response is React! That is the correct answer. React is a modern frontend library, but PyTorch and Tensorflow are not.
 
+.. image:: https://static.scarf.sh/a.png?x-pxid=51f549d1-aabf-473c-b971-f8d9c3ac8ac5
```

Comment:
We are experimenting with [Scarf.sh](https://about.scarf.sh/documentation-insights). It is a privacy-friendly analytics product. Please let me know if you have any concerns about it.

Reply:
Thanks for keeping me updated, I think it's fine.